projects/plugins/jetpack/modules/likes.php

```diff
@@ -388,32 +403,34 @@ function enqueue_admin_scripts() {
 						'modules/likes/post-count-jetpack.js'
 					),
 					array( 'likes-post-count' ),
-					JETPACK__VERSION
+					JETPACK__VERSION,
+					$in_footer = false
 				);
 			} else {
-				wp_enqueue_script( 'jquery.wpcom-proxy-request', "/wp-content/js/jquery/jquery.wpcom-proxy-request.js", array('jquery'), NULL, true );
-				wp_enqueue_script( 'likes-post-count', plugins_url( 'likes/post-count.js', dirname( __FILE__ ) ), array( 'jquery' ), JETPACK__VERSION );
-				wp_enqueue_script( 'likes-post-count-wpcom', plugins_url( 'likes/post-count-wpcom.js', dirname( __FILE__ ) ), array( 'likes-post-count', 'jquery.wpcom-proxy-request' ), JETPACK__VERSION );
+				wp_enqueue_script( 'jquery.wpcom-proxy-request', '/wp-content/js/jquery/jquery.wpcom-proxy-request.js', array( 'jquery' ), JETPACK__VERSION, true );
+				wp_enqueue_script( 'likes-post-count', plugins_url( 'likes/post-count.js', dirname( __DIR__ ) ), array( 'jquery' ), JETPACK__VERSION, $in_footer = false );
+				wp_enqueue_script( 'likes-post-count-wpcom', plugins_url( 'likes/post-count-wpcom.js', dirname( __DIR__ ) ), array( 'likes-post-count', 'jquery.wpcom-proxy-request' ), JETPACK__VERSION, $in_footer = false );
```

Comment:
Changing `dirname( __FILE__ )` to `dirname( __DIR__ )` is not equivalent. You probably meant to change to just `__DIR__`.
```suggestion
				wp_enqueue_script( 'likes-post-count', plugins_url( 'likes/post-count.js', __DIR__ ), array( 'jquery' ), JETPACK__VERSION, $in_footer = false );
				wp_enqueue_script( 'likes-post-count-wpcom', plugins_url( 'likes/post-count-wpcom.js', __DIR__ ), array( 'likes-post-count', 'jquery.wpcom-proxy-request' ), JETPACK__VERSION, $in_footer = false );
```
Or was this code broken before and you're now fixing it?

Reply:
> You probably meant to change to just __DIR__.

Nice catch! That one was on me.